### PR TITLE
RFC-0003: Member Eligibility Criteria

### PR DESCRIPTION
**Pull request by beeshot**
A first draft of RFC 0003

### CODE REVIEW

File: governance/0003-onboarding-governance.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+>   * A non-governmental organization (NGO) or Social Purpose Organization with atleast five (5) years operating history.
+>   * An accredited university or other institution of higher education with at least five(5) years operating history.
+>   * A certificate authority (CA) with at least five (5) years operating history.
+
+Bloxberg regulates membership requirements in its [Bloxberg Whitepaper v1.1](https://bloxberg.org/wp-content/uploads/2020/02/bloxberg_whitepaper_1.1.pdf) as follows:
+
+> An Institution can apply as a bloxberg consortium member, if it is a ‘Science organization, specifically academic, higher education and primarily publically funded research institutions.
+
+
+Sovrin regulates membership requirements in its [Sovrin Business Policy V2 Policy](https://sovrin.org/wp-content/uploads/Sovrin-Steward-Business-Policies-V2.pdf) as follows:
```
[the remark this answers is not in the document]
Comment (chinmay241) [reply governance/0003-onboarding-governance.md]: I had to re-quote source text since it answered specific questions. 
Hence I was re-quoting specific answers from the text it was derived.

[the remark this answers is not in the document]
Comment (kiview) [reply governance/0003-onboarding-governance.md]: This is fine, but the introductory sentence should, therefore, be more specific, since the following quote is not talking about membership requirements in general, but e.g. requirements are suspension.

[the remark this answers is not in the document]
Comment (chinmay241) [reply governance/0003-onboarding-governance.md]: I have fixed this ^

**Review by fflush — APPROVED** [review]
Looks very good. I have no suggestions for improvement. Fits for me so.

**Comment by andionita** [comment]
I think it looks pretty sharp now. Good job editing it!

**Review by andionita — APPROVED** [review]

File: governance/0003-onboarding-governance.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Clearly defined criteria for eligibility of members will increase security, as well as transparency within the community and parties that can participate in the blockchain network.
+Moreover, they can increase trust within the network, especially in the beginning, when a decentralized network is more prone to security risks since a malicious member can have a potentially higher impact.
+
+## Specifications
+
+* Organization forms which can join the consortium.
+* The jurisdiction, the member has to be compliant with.
+* Requirements for organizations/companies to join.
+* Joining without meeting the requirements.
+* Special requirements for suspended and terminated members.
```
Comment (kiview) [inline governance/0003-onboarding-governance.md:25]:
```suggestion
* Special requirements for suspended and terminated members that wish to be reinstantiated.
```

File: governance/0003-onboarding-governance.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+* If the criteria are defined too narrow for a small pool of possible members, potentially interested and interesting members may be excluded. 
+Therefore the targeted size of the network and the size of the set of potential members should be taken into consideration.
+
+## Prior art
+
+* The [Sovrin Business Policy V2 Policy](https://sovrin.org/wp-content/uploads/Sovrin-Steward-Business-Policies-V2.pdf) and [Bloxberg Whitepaper v1.1](https://bloxberg.org/wp-content/uploads/2020/02/bloxberg_whitepaper_1.1.pdf) has been used to retrieve possible answers within this document. 
+
+## Unresolved questions
+
+* When is an application considered complete for review?
```
Comment (kiview) [inline governance/0003-onboarding-governance.md:98]: This line seems to be unrelated to this RFC.

**Review by kiview — APPROVED** [review]
LGTM, made just 2 additional comments.
